cloudplatform/connectivity-destination-service/src/main/java/com/sap/cloud/sdk/cloudplatform/connectivity/DestinationKeyStoreExtractor.java

```diff
@@ -43,10 +46,11 @@ class DestinationKeyStoreExtractor
     // See the supported key store file extensions:
     // https://help.sap.com/viewer/cca91383641e40ffbe03bdc78f00f681/Cloud/en-US/df1bb55a526942b9bee78fea2ebb3162.html
     //Mapping file extension to key store types
-    private static final Map<String, String> SUPPORTED_KEY_STORE_TYPES_AS_KEY_STORE = ImmutableMap.of(
-            "pfx", "PKCS12",
-            "p12", "PKCS12",
-            "jks", "JKS"
+    private static final Map<String, BiFunction<String, String, KeyStore>> SUPPORTED_KEY_STORES = ImmutableMap.of(
+        "pfx", (cert,pw) -> retrieveExistingKeyStore(cert, pw, "PKCS12"),
```

Comment:
```suggestion
        "pfx", (ks,pw) -> retrieveExistingKeyStore(ks, pw, "PKCS12"),
```

(minor) just an idea to clarify what these strings represent for the 3 keystore cases..